server/dba/migrations/20-forests.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+module.exports = {
+  up: function(queryInterface, Sequelize) {
+    return queryInterface.createTable('forests', {
+      id: { type: Sequelize.INTEGER, primaryKey: true, field: 'id' },
+      forestName: { type: Sequelize.STRING(50), field: 'forest_name' },
+      description: { type: Sequelize.STRING(1000), field: 'description' },
```

Comment:
I don't see where this description is on the frontend?

Reply:
@lauraGgit correct, description field not being used. We'll drop once we are set with all the content.

Reply:
@lpendem - could you please let me know what story/ task that action has been documented in?

Reply:
@lauraGgit this filed is being using to store forest key works, forestName, city, state etc in story: https://github.com/nciinc/fs-intake-module/issues/5